src/types/secrets.ts

```diff
@@ -1,8 +1,3 @@
 import {Secrets as GSecrets} from 'src/client'
-import {NormalizedState} from 'src/types/resources'
 
 export type Secret = GSecrets
```

Comment:
the GSecret type is super vague. Can we update the API contract to return something that's more valid than `any`? Do we want to do that or does it leave us vulnerable?